addons/web/static/tests/core/misc_tests.js

```diff
@@ -0,0 +1,92 @@
+odoo.define('web.misc_tests', function (require) {
+"use strict";
+
+const core = require('web.core');
+const AbstractAction = require('web.AbstractAction');
+const Notification = require('web.Notification');
+const NotificationService = require('web.NotificationService');
+
+const testUtils = require('web.test_utils');
+const createActionManager = testUtils.createActionManager;
+
+const waitCloseNotification = function () {
+    return new Promise(function (resolve) {
+        setTimeout(resolve, 1);
+    });
+};
+
+QUnit.module('core', {
+    beforeEach: function () {
+        // We need to use a delay above 0 ms because otherwise the notification will close right after it opens
+        // before we can perform any test.
+        testUtils.mock.patch(Notification, {
+            _autoCloseDelay: 1,
+            _animation: false,
+        });
+    },
+    afterEach: function () {
+        // The Notification Service has a side effect: it adds a div inside
+        // document.body.  We could implement a cleanup mechanism for services,
+        // but this seems a little overkill since services are not supposed to
+        // be destroyed anyway.
+        $('.o_notification_manager').remove();
+        testUtils.mock.unpatch(Notification);
+    }
```

Comment:
also, i'd simply add a test in action_manager_tests (this concerns the action_manager, even though the function is define in misc, and in this file, you already have a lot of required elements correctly set up)